### ISSUE TITLE
Deprecated API warning: Calls to RED.util.evaluateJSONataExpression must include a callback

### ISSUE THREAD

**Issue by zachowj** [jensrossbach/node-red-contrib-chronos#116]
**Describe the bug**
```
nodered  | 3 Oct 18:05:52 - [warn] Deprecated API warning: Calls to RED.util.evaluateJSONataExpression must include a callback. This will not be optional in Node-RED 4.0. Please identify the node from the following stack and check for an update on npm. If none is available, please notify the node author.
nodered  | 3 Oct 18:05:52 - [warn] Error
nodered  |     at Object.evaluateJSONataExpression (/usr/src/node-red/node_modules/@node-red/util/lib/util.js:775:18)
nodered  |     at getJSONataValue (/data/node_modules/node-red-contrib-chronos/nodes/scheduler.js:761:33)
nodered  |     at Timeout.handleTimeout [as _onTimeout] (/data/node_modules/node-red-contrib-chronos/nodes/scheduler.js:707:37)
nodered  |     at listOnTimeout (node:internal/timers:571:11)
nodered  |     at process.processTimers (node:internal/timers:512:7)
```

**To reproduce**
1. Drop in a scheduler node
2. Change the schedule output to use JSONata

**Expected behavior**

No warning about deprecated API

**Screenshots**

![image](https://github.com/jensrossbach/node-red-contrib-chronos/assets/37859597/037cd003-1b32-4823-ada2-3ef4b5e19de8)

**Context information**
Please complete the following information.
 - Platform & OS: NR docker image
 - node-red-contrib-chronos version: 1.19.0
 - Node-RED version: 3.1.0
 - Node.js version: 18.17.1

**Additional information**

**Comment by jensrossbach** [comment]
This may take a while, it's not an easy fix as the behavior is changed fudamentally (from synchronous to asynchronous).

**Comment by mickym2** [comment]
So I noticed the same error - and have detected these nodes recently. - So let me say you many thanks to your idea for these nodes and your work you have done until now. I will replace most of my other scheduling nodes, where I used no JSONATA. But with JSONATA this error floods the debug window. Thanks for your effort up to now!

**Comment by jensrossbach** [comment]
Thank you @mickym2! If the error messages are really flooding the logs, I will check if I can maybe release a beta version. I'm not sure if NPM/Node-RED supports pre-releases but let's see.

**Comment by mickym2** [comment]
This is not so important, as the logs are OK, if you don't use JSONATA output. In case JSONATA is needed - I just added a change node after the scheduler node as a workaround. So no problem anymore at the moment.

**Comment by jensrossbach** [comment]
Meanwhile I decided to publish the fix already as I cannot predict when I will be finished with unit test adaptations. I disabled unit tests in the QA workflow for the time being and will update them in background.

So this issue is fixed with release [1.19.1](https://github.com/jensrossbach/node-red-contrib-chronos/releases/tag/v1.19.1).
